OpenRA.Mods.Common/Traits/LevelupWhenCreated.cs

```diff
@@ -0,0 +1,47 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Actors possessing this trait should define the GainsExperience trait beforehand. When the prerequisites are fulfilled, ",
```

Comment:
@penev92, isn't `Requires<Trait>` only needed when `Trait` is used in the constructor?

I'll remove the "beforehand" from the comment and add change `.TraidOrDefault()` to `.Trait()`, but give me a confirmation for the `Requires<>` pls.